.github/workflows/build.yml

```diff
@@ -356,22 +300,18 @@ jobs:
     - name: Configure SSL lib env vars
       run: |
         echo "MULTISSL_DIR=${GITHUB_WORKSPACE}/multissl" >> "$GITHUB_ENV"
-        echo "OPENSSL_DIR=${GITHUB_WORKSPACE}/multissl/aws-lc/${AWSLC_VER}" >> "$GITHUB_ENV"
-        echo "LD_LIBRARY_PATH=${GITHUB_WORKSPACE}/multissl/aws-lc/${AWSLC_VER}/lib" >> "$GITHUB_ENV"
-    - name: 'Restore AWS-LC build'
-      id: cache-aws-lc
+        echo "SSL_DIR=${GITHUB_WORKSPACE}/multissl/${{ matrix.ssl }}/${SSL_VER}" >> "$GITHUB_ENV"
```

Comment:
Can't we just use `${{ env.ssldir }}`?  I don't understand why we're defining it here and in the `env` section

Reply:
zizmor says using `${{ env.ssldir }}` can result in [template injection](https://docs.zizmor.sh/audits/#template-injection):

```
help[template-injection]: code injection via template expansion
   --> ./.github/workflows/build.yml:324:134
    |
323 |       run: |
    |       --- help: this run block
324 |         CMD=(./configure CFLAGS="-fdiagnostics-format=json" --config-cache --enable-slower-safety --with-pydebug --with-openssl="${{ env.ssldir }...
    |                                                                                                                                      ---------- help: may expand into attacker-controllable code
```

Reply:
Isn't `env` a procteced namespace that cannot be accessed from outside?

Reply:
Let's ask zizmor author @woodruffw.

Reply:
Ok, so zizmor kwnows that it's not attackable *but* recommends against it for shell expansion purposes: https://github.com/woodruffw/gha-hazmat/blob/e4094ceabb0068a2a5b81ecc8e1ee919ba52a050/.github/workflows/template-injection.yml#L77.

Reply:
Some more docs here too (the first tip in that section mentions why `env.*` is not preferred versus direct expansion): https://docs.zizmor.sh/audits/#template-injection